src/components/Article.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CommentList from "./CommentList";
+
+class Article extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            isOpen: true
+        };
+        this.handleButtonClick = this.handleButtonClick.bind(this);
+    }
+
+    handleButtonClick = () => {
+        this.setState({
+            isOpen: !this.state.isOpen
+        })
+    };
+
+    render() {
+        let body = '';
```

Comment:
use const where it's possible (i.e. everywhere)

Reply:
Done.